tests/packagedcode/data/buildpack/paketo-buildpacks/opentelemetry/buildpack.toml

```diff
@@ -0,0 +1,65 @@
+# Copyright 2018-2024 the original author or authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+api = "0.7"
+
+[buildpack]
+  description = "A Cloud Native Buildpack that contributes and configures the OpenTelemetry Agent"
+  homepage = "https://github.com/paketo-buildpacks/opentelemetry"
+  id = "paketo-buildpacks/opentelemetry"
+  keywords = ["java", "apm", "trace", "opentelemetry"]
+  name = "Paketo Buildpack for OpenTelemetry"
+  sbom-formats = ["application/vnd.cyclonedx+json", "application/vnd.syft+json"]
+  version = "{{.version}}"
```

Comment:
This is not useful to parse and store directly as version, check the spec for how we could deduce this